bitnami/matomo/templates/cronjob.yaml

```diff
@@ -0,0 +1,261 @@
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  schedule: {{ .Values.cronjobs.archive.schedule | quote }}
+  suspend: {{ .Values.cronjobs.archive.suspend }}
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels: {{- include "common.labels.standard" . | nindent 12 }}
+            app.kubernetes.io/component: archive
+            {{- if .Values.cronjobs.archive.podLabels }}
+            {{- include "common.tplvalues.render" ( dict "value" .Values.cronjobs.archive.podLabels "context" $ ) | nindent 12 }}
+            {{- end }}
+          {{- if .Values.cronjobs.archive.podAnnotations }}
+          annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.cronjobs.archive.podAnnotations "context" $ ) | nindent 12 }}
+          {{- end }}
+        spec:
+          {{- include "matomo.imagePullSecrets" . | nindent 10 }}
+          restartPolicy: OnFailure
+          initContainers:
+            {{ include "matomo.initContainers" . | nindent 12 }}
+          containers:
+            - name: {{ include "common.names.fullname" . }}-scheduled-tasks
+              image: {{ template "matomo.image" . }}
+              imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+              {{- if .Values.cronjobs.archive.containerSecurityContext.enabled }}
+              securityContext: {{- omit .Values.cronjobs.archive.containerSecurityContext "enabled" | toYaml | nindent 16 }}
+              {{- end }}
+              {{- if .Values.cronjobs.archive.command }}
+              command: {{- include "common.tplvalues.render" (dict "value" .Values.cronjobs.archive.command "context" $) | nindent 16 }}
+              {{- else }}
+              command:
+                - /bin/bash
+                - -c
+              {{- end }}
+              {{- if .Values.cronjobs.archive.args }}
+              args: {{- include "common.tplvalues.render" (dict "value" .Values.cronjobs.archive.args "context" $) | nindent 16 }}
+              {{- else }}
+              args:
+                - . /opt/bitnami/scripts/matomo-env.sh &&
+                  . /opt/bitnami/scripts/libbitnami.sh &&
+                  . /opt/bitnami/scripts/liblog.sh &&
+                  /opt/bitnami/scripts/php/setup.sh &&
+                  /opt/bitnami/scripts/mysql-client/setup.sh &&
+                  /opt/bitnami/scripts/matomo/setup.sh &&
+                  /post-init.sh &&
+                  /opt/bitnami/php/bin/php /opt/bitnami/matomo/console core:run-scheduled-tasks
```

Comment:
Feel free to keep it as it is but I'd include new scripts in the container image (e.g. `/opt/bitnami/scripts/matomo/scheduled-tasks.sh` &  `/opt/bitnami/scripts/matomo/archive.sh`) and execute them from here.

Reply:
IMHO I cannot see a huge advantage in case of having the `.sh`'s integrated in the container, since they would become a snippet to maintain with pretty much the includes of the entrypoint, plus the fact once inside the running container, the users could just run both with the single commands:

`/opt/bitnami/php/bin/php /opt/bitnami/matomo/console core:run-scheduled-tasks` 
`/opt/bitnami/php/bin/php /opt/bitnami/matomo/console core:archive` 

Nevertheless, I would indeed see the advantage of having the `run.sh` passthrough as most of the bitnami containers as mentioned before, but I am not sure of the real need of it apart of this case (matomo has a defined passthrough to one or another webserver rn instead the common one)

I would go with leaving this snippets directly on the jobs as in other assets, for now.